component/processor.go

```diff
@@ -24,10 +24,7 @@ import (
 // ProcessorConfig is the configuration of a component.Processor. Specific extensions must implement
```

Comment:
```suggestion
// ProcessorConfig is the configuration of a component.Processor. Specific processors must implement
```

Reply:
Done separately since not related to this PR. https://github.com/open-telemetry/opentelemetry-collector/pull/6551